docs/json_rpc_api.md

```diff
@@ -632,12 +632,12 @@ The following is an overview of the RPC methods which are implemented by btcd, b
 |   |   |
 |---|---|
 |Method|searchrawtransactions|
-|Parameters|1. address (string, required) - bitcoin address <br /> 2. verbose (int, optional, default=true) - specifies the transaction is returned as a JSON object instead of hex-encoded string <br />3. skip (int, optional, default=0) - the number of leading transactions to leave out of the final response <br /> 4. count (int, optional, default=100) - the maximum number of transactions to return|
+|Parameters|1. address (string, required) - bitcoin address <br /> 2. verbose (int, optional, default=true) - specifies the transaction is returned as a JSON object instead of hex-encoded string <br />3. skip (int, optional, default=0) - the number of leading transactions to leave out of the final response <br /> 4. count (int, optional, default=100) - the maximum number of transactions to return <br /> 5. (vinextra numeric, optional, default=0) - Specify that extra data from previous output will be returned in vin|

```

Comment:
done.

fwiw, I had it the way it was because i copied the exact output from btcctl help searchrawtransactions, which prints numeric instead of int.   ie:

<pre>
Arguments:
1. address  (string, required)               The Bitcoin address to search for
2. verbose  (numeric, optional, default=1)   Specifies the transaction is returned as a JSON object instead of hex
-encoded string
3. skip     (numeric, optional, default=0)   The number of leading transactions to leave out of the final response
4. count    (numeric, optional, default=100) The maximum number of transactions to return
5. vinextra (numeric, optional, default=0)   Specify that extra data from previous output will be returned in vin
</pre>